app/models/customer.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+from app import db
+# from datetime import datetime
+from sqlalchemy.orm import backref, relationship
+from sqlalchemy.sql.schema import ForeignKey
+from sqlalchemy import Table, Column, Integer, ForeignKey 
+
+# from flask_migrate import Migrate
+
+
+class Customer(db.Model):
+    __tablename__ = "customers"
+
+    customer_id = db.Column(db.Integer, primary_key=True,autoincrement=True)
+    name = db.Column(db.String)    
+    register_at = db.Column(db.DateTime, nullable=True) # nullable=True ??? 
+    postal_code = db.Column(db.String) 
+    phone = db.Column(db.String) 
+    videos_checked_out_count =  db.Column(db.Integer, default=0)
+    
+    def get_json(self):
+        return {
+            "id":self.customer_id,
+            "name":self.name,
+            "registered_at":self.register_at,
+            "postal_code":int(self.postal_code),
+            "phone":self.phone,
+            "videos_checked_out_count":self.videos_checked_out_count
```

Comment:
Once you have the relationship the `videos_checked_out_count` is just the length of the `rentals` that haven't been checked in:

```suggestion
            "videos_checked_out_count": len([rental for rental in self.rentals if not rental.check_in_date])
```